scripts/app-version-get

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+set -euo pipefail # exit on error; treat unset variables as errors; exit on errors in piped commands
+git describe --exact-match --tags 2>/dev/null || (echo -n "dev-" && git rev-parse --short HEAD) # e.g. "v2.2" or "dev-a821087"
```

Comment:
How would you end up with `v2.2`? From git tags?

Reply:
Yes! The GitHub release process that we use creates them automatically.